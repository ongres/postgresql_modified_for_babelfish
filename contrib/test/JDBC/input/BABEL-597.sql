-- create procedure with incomplete definition
create procedure [p_employee_insert]
@person_id int, @fname varchar(20), @lname varchar(30), @sal money
as
begin
go
