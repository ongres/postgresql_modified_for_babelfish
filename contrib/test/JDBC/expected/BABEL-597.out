-- create procedure with incomplete definition
create procedure [p_employee_insert]
@person_id int, @fname varchar(20), @lname varchar(30), @sal money
as
begin
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: syntax error near '<EOF>' at line 6 and character position 0)~~

